utils/createJestConfig.js

```diff
@@ -14,16 +14,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
     : undefined;
 
   const config = {
-    roots: ['<rootDir>/src'],
+    roots: ['<rootDir>/src', '<rootDir>/test', '<rootDir>/tests'],
 
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
+    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', 'tests?/**/*.{js,jsx,ts,tsx}','!src/**/*.d.ts'],
```

Comment:
Should you have something like "!tests?/**/*.d.ts'"?

Reply:
done